packages/core/core/src/utils/transform-content-types-to-models.ts

```diff
@@ -317,6 +317,19 @@ export const transformContentTypesToModels = (
       },
     };
 
+    // Add indexes to model
+    if (contentType.modelType === 'contentType') {
+      model.indexes = [
```

Comment:
spread model.indexes in it in case a user have manually defined some ? (not sure we support it still/yet)

Reply:
We don't yet be we really should